internal/common/utils.go

```diff
@@ -4,11 +4,45 @@ import (
 	"bytes"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"strings"
 
 	"github.com/go-git/go-git/v5"
 )
 
+const GittufNamespace = "refs/gittuf"
+
+// InitializeGittufNamespace creates the refs/gittuf directory if it does not
+// already exist. It does not initialize the refs for policy and RSL namespaces.
+func InitializeGittufNamespace(targetDir string) error {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	if err := os.Chdir(targetDir); err != nil {
+		return err
+	}
+	defer os.Chdir(cwd) //nolint:errcheck
```

Comment:
This is fine as is, but if you want in the future [`--work-dir` flag](https://git-scm.com/docs/git#Documentation/git.txt---work-treeltpathgt) will let you run commands from another folder.